src/client/utils/serialise.py

```diff
@@ -2,8 +2,30 @@
 
 
 class Serialiser:
```

Comment:
Looks good in general.

- We need to import pickle? 
- It might be good to implement error handling.

Reply:
yes we do need pickle for serialise and deserialise